scripts/dataGenerator.js

```diff
@@ -343,12 +352,15 @@ var createQuestionsTrueFalse = function () {
  * @param {integer} id
  */
 var addQuestionTrueFalse = function (qTopic, id) {
```

Comment:
I know you didn't make the functions for the questions (ex addQuestionTrueFalse) but do you think it'd be possible to have just one question function and then have another function to change the type of each question so we have less duplicated code?

@esteeroramy @mehtadarshan @ahmedqarmout @larryyueli @Mohammed-JS-Khan

Reply:
Well, this is the Datagenerator so efficiency here is not our main concern. The issue is that the processes are async, and so you can only add a type of question, answer it, then add the next one, etc... And so with this design each function waits on the one before it in order to run